style.css

```diff
@@ -4,23 +4,12 @@
     font-family: sans-serif;
 }
 
-body {
-    display: flex;
```

Comment:
The browser's default layout mode is "block layout," which renders elements vertically from top to bottom. Using `display: flex` for a simple column layout is unnecessary unless you need specific flexbox features.